Kwc/Box/TitleEditable/Trl/Component.php

```diff
@@ -1,15 +1,56 @@
 <?php
-class Kwc_Box_TitleEditable_Trl_Component extends Kwc_Chained_Trl_MasterAsChild_Component
+class Kwc_Box_TitleEditable_Trl_Component extends Kwc_Box_Title_Trl_Component
 {
     public static function getSettings($masterComponentClass)
     {
         $ret = parent::getSettings($masterComponentClass);
         $ret['flags']['hasHeaderIncludeCode'] = true;
+        $ret['ownModel'] = 'Kwf_Component_FieldModel';
         return $ret;
     }
 
     public function getIncludeCode()
     {
         return $this->getData();
     }
+
+    protected function _getTitle()
```

Comment:
This function is a collection from Kwc_Box_TitleEditable_Component::_getTitle and Kwc_Box_Title_Component::_getTitle(). Maybe this should be solved differently...